### ISSUE TITLE
Run docker with the default MountFlags makes the wrong mount table in the centos7

### ISSUE THREAD

**Issue by BrickXu** [moby/moby#27676]
**Description**

After upgrade docker daemon to 1.12, start containers will append the mount info,for example:

```
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda5        10G  1.8G  8.2G  19% /
devtmpfs         63G     0   63G   0% /dev
tmpfs            63G   12K   63G   1% /dev/shm
tmpfs            63G  515M   63G   1% /run
tmpfs            63G     0   63G   0% /sys/fs/cgroup
/dev/sda7       2.0G   34M  2.0G   2% /home
/dev/sda3        20G 1023M   19G   5% /var
/dev/sda1       492M  128M  365M  26% /boot
/dev/sda8       877G  306M  877G   1% /home/q
/dev/dm-3        10G  837M  9.2G   9% /home/q/docker/devicemapper/mnt/1794abdbce1d8669bed0d78e41878c79b0cb7ce6b370f7beb860668f2c468f38
shm              64M     0   64M   0% /home/q/docker/containers/3873602c813c556e6dfc08f4fb3c306a06c9474196675d60075ddc135b5ed260/shm
/dev/dm-4        10G   62M   10G   1% /home/q/docker/devicemapper/mnt/efb2967e19cf1ae8a54ad597e52e1ff477a670a001b9f6b0a94162eee6bb1d3a
shm              64M     0   64M   0% /home/q/docker/containers/5d3f9bdfe38408148ec0615e7580755d8c47390ad00d39600dcbe1ed62723732/shm
/dev/dm-8        10G  548M  9.5G   6% /home/q/docker/devicemapper/mnt/39236ad75da7ba2b56db24576a0faf3ccfa792fe0891ba160dbb493f97b0377a
shm              64M     0   64M   0% /home/q/docker/containers/6f166a7311a4225014fcfcd3ea036f498a1d0aa4b0dfef3a0c877a91d8df9137/shm
/dev/dm-7        10G  548M  9.5G   6% /home/q/docker/devicemapper/mnt/14e1ba5ce910964f145d963d2a77260d6a0e1cce41a0f7d46871afd11386b427
shm              64M     0   64M   0% /home/q/docker/containers/1ca57ef76a2709591514b1ff18360161c2c4bd576856224d6044337568475529/shm
/dev/dm-11       10G  1.3G  8.8G  13% /home/q/docker/devicemapper/mnt/c2b9f1393e02e5007f85767899ef8a5d8955061c7ac0b752bdeed4efd4b72c49
shm              64M     0   64M   0% /home/q/docker/containers/c6d7e0aaaac0c162fd2d8b1a4cfc5e56f6ccc4329034c156acad2c513ad1427e/shm
/dev/dm-9        10G  548M  9.5G   6% /home/q/docker/devicemapper/mnt/77da72b9e27365e2ef43ab2401db9fc4100c517f0823b28eea99af2ef132c710
shm              64M     0   64M   0% /home/q/docker/containers/c8d5d740c067c0130623bf279a1690372dde9eb38919174a619991cc49ea377f/shm
/dev/dm-5        10G  827M  9.2G   9% /home/q/docker/devicemapper/mnt/a07a9fddcb5bac3db044876fb2d4b955a4aa6361939ebe1b6322c0ba103bab7c
shm              64M     0   64M   0% /home/q/docker/containers/ad482fe17006ef6b0bb498cd5440d12618db621de308ee4b482ffd87dbf63244/shm
/dev/dm-12       10G  827M  9.2G   9% /home/q/docker/devicemapper/mnt/5e52e07a37e11f2270ed8688a455010f9678a1db6179df5fbc86dcc919071487
shm              64M     0   64M   0% /home/q/docker/containers/14ed512689faa06e1d704711be261247bf0c299f81e3d31f81f122b7e4e2366f/shm
/dev/dm-13       10G  765M  9.3G   8% /home/q/docker/devicemapper/mnt/e8ce7dd6b9dd76af66dc51e62667e794a68b673210168c8d60d11a63c475db5c
shm              64M     0   64M   0% /home/q/docker/containers/7962e183a85c2945a6728fa43a31e414000875d5aae7ef06665fd9cdb83b2d6e/shm
/dev/dm-14       10G  765M  9.3G   8% /home/q/docker/devicemapper/mnt/72fe4cfd2fd77a7e34cbb4e8fc05f2e87b1a23c622216f13b8fb2fd689e1ed78
shm              64M     0   64M   0% /home/q/docker/containers/d40429aaf51fef75aae591ec92e14178607a6fc518621044f2bb27e15122761c/shm
/dev/dm-15       10G  765M  9.3G   8% /home/q/docker/devicemapper/mnt/53b2ce3964a57539d1e8f2443282b99d0edb5fe060604341cd6820a100b83a3b
shm              64M     0   64M   0% /home/q/docker/containers/dedda6b6e3719d22e4783141947824563e081d88985de11ad23b78d56ede3e77/shm
/dev/dm-16       10G  765M  9.3G   8% /home/q/docker/devicemapper/mnt/9b05251aa0f2ac4d4065eff223d86765d2307edc09efe405042bba966fac957c
shm              64M     0   64M   0% /home/q/docker/containers/ffb85d0a30969572f717b28dc7e4e218d95c690a50357d7f7a02dfc80ef1266b/shm
/dev/dm-17       10G  548M  9.5G   6% /home/q/docker/devicemapper/mnt/3da2f647032aa735368c4fb2637e8988cd1b1ad047f930711eace948a0ad84df
shm              64M     0   64M   0% /home/q/docker/containers/dee7ca45fd43f47542ffc2f4a0e80c531a3aa9fed3ce4325f39df8d7b80cf671/shm
/dev/dm-10       10G  234M  9.8G   3% /home/q/docker/devicemapper/mnt/f7725e519c132d596de0abd6af6c7c74a91de1fd5c38e9345186ed02eacb8c43
shm              64M     0   64M   0% /home/q/docker/containers/7c5b33460a0aa63f889eaeb7a9c06a7d8eabd5de4b36b90696d1eb7644eeb607/shm
tmpfs            13G     0   13G   0% /run/user/30305
```

And we use `nrpe` to monitor the disk usage, but it raise an error because the volumes that docker mounted.

```
DISK CRITICAL - net:[4026531968] is not accessible: No such file or directory
```

Is there any issue or PR to explain why do this in the 1.12? Could I disable this behavior ?

**Output of `docker version`:**

```
Client:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        
 OS/Arch:      linux/amd64
```

**Output of `docker info`:**

```
Containers: 20
 Running: 14
 Paused: 0
 Stopped: 6
Images: 17
Server Version: 1.12.1
Storage Driver: devicemapper
 Pool Name: docker_vg-thinpool
 Pool Blocksize: 524.3 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: 
 Metadata file: 
 Data Space Used: 12.89 GB
 Data Space Total: 171.8 GB
 Data Space Available: 158.9 GB
 Metadata Space Used: 6.5 MB
 Metadata Space Total: 10.73 GB
 Metadata Space Available: 10.73 GB
 Thin Pool Minimum Free Space: 17.18 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: true
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Library Version: 1.02.107-RHEL7 (2016-06-09)
Logging Driver: none
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: overlay null host bridge
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: seccomp
Kernel Version: 3.10.0-327.36.2.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
CPUs: 24
Total Memory: 125.8 GiB
Name: ****
ID: VYLP:W2DW:OCGJ:6YBH:6FBW:RTHJ:2KTF:Y3K3:UVZO:2MFD:RASN:HZM7
Docker Root Dir: /home/q/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
Labels:
Insecure Registries:
```

**Comment by thaJeztah** [comment]
Possibly related to https://github.com/docker/docker/pull/22806, however, I'm not sure nrpe should be checking these mounts. Having them excluded would probably be a better option than changing dockeri

**Comment by BrickXu** [comment]
Hi @thaJeztah , thanks for your response, I try to ignore or exclude the path in the check_disk, it does not work, I'm still working on it to find out the root cause.

**Comment by BrickXu** [comment]
Hi @thaJeztah, I think it's the problem about #22806, that set `MountFlags` to the default value, here is some example:

### MountFlags=shared

The following is the original context in the `/etc/mtab`:

```
rootfs / rootfs rw 0 0
sysfs /sys sysfs rw,nosuid,nodev,noexec,relatime 0 0
proc /proc proc rw,nosuid,nodev,noexec,relatime 0 0
devtmpfs /dev devtmpfs rw,nosuid,size=65946932k,nr_inodes=16486733,mode=755 0 0
securityfs /sys/kernel/security securityfs rw,nosuid,nodev,noexec,relatime 0 0
tmpfs /dev/shm tmpfs rw,nosuid,nodev 0 0
devpts /dev/pts devpts rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000 0 0
tmpfs /run tmpfs rw,nosuid,nodev,mode=755 0 0
tmpfs /sys/fs/cgroup tmpfs ro,nosuid,nodev,noexec,mode=755 0 0
cgroup /sys/fs/cgroup/systemd cgroup rw,nosuid,nodev,noexec,relatime,xattr,release_agent=/usr/lib/systemd/systemd-cgroups-agent,name=systemd 0 0
pstore /sys/fs/pstore pstore rw,nosuid,nodev,noexec,relatime 0 0
cgroup /sys/fs/cgroup/cpu,cpuacct cgroup rw,nosuid,nodev,noexec,relatime,cpuacct,cpu 0 0
cgroup /sys/fs/cgroup/cpuset cgroup rw,nosuid,nodev,noexec,relatime,cpuset 0 0
cgroup /sys/fs/cgroup/devices cgroup rw,nosuid,nodev,noexec,relatime,devices 0 0
cgroup /sys/fs/cgroup/perf_event cgroup rw,nosuid,nodev,noexec,relatime,perf_event 0 0
cgroup /sys/fs/cgroup/hugetlb cgroup rw,nosuid,nodev,noexec,relatime,hugetlb 0 0
cgroup /sys/fs/cgroup/net_cls cgroup rw,nosuid,nodev,noexec,relatime,net_cls 0 0
cgroup /sys/fs/cgroup/blkio cgroup rw,nosuid,nodev,noexec,relatime,blkio 0 0
cgroup /sys/fs/cgroup/memory cgroup rw,nosuid,nodev,noexec,relatime,memory 0 0
cgroup /sys/fs/cgroup/freezer cgroup rw,nosuid,nodev,noexec,relatime,freezer 0 0
configfs /sys/kernel/config configfs rw,relatime 0 0
/dev/sda5 / xfs rw,relatime,attr2,inode64,noquota 0 0
systemd-1 /proc/sys/fs/binfmt_misc autofs rw,relatime,fd=33,pgrp=1,timeout=300,minproto=5,maxproto=5,direct 0 0
mqueue /dev/mqueue mqueue rw,relatime 0 0
debugfs /sys/kernel/debug debugfs rw,relatime 0 0
hugetlbfs /dev/hugepages hugetlbfs rw,relatime 0 0
/dev/sdb1 /data1 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdd1 /data3 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdf1 /data5 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdg1 /data6 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdc1 /data2 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sde1 /data4 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda3 /var xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda7 /home xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda1 /boot xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda8 /home/q xfs rw,relatime,attr2,inode64,noquota 0 0
binfmt_misc /proc/sys/fs/binfmt_misc binfmt_misc rw,relatime 0 0
tmpfs /run/user/30303 tmpfs rw,nosuid,nodev,relatime,size=13191248k,mode=700,uid=30303,gid=1001 0 0
/dev/sda8 /home/q/docker/devicemapper xfs rw,relatime,attr2,inode64,noquota 0 0
```
then I start a container, ubuntu:14.04, I found here is a unusual mount point in the file(at the last of the file):

```
rootfs / rootfs rw 0 0
sysfs /sys sysfs rw,nosuid,nodev,noexec,relatime 0 0
proc /proc proc rw,nosuid,nodev,noexec,relatime 0 0
devtmpfs /dev devtmpfs rw,nosuid,size=65946932k,nr_inodes=16486733,mode=755 0 0
securityfs /sys/kernel/security securityfs rw,nosuid,nodev,noexec,relatime 0 0
tmpfs /dev/shm tmpfs rw,nosuid,nodev 0 0
devpts /dev/pts devpts rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000 0 0
tmpfs /run tmpfs rw,nosuid,nodev,mode=755 0 0
tmpfs /sys/fs/cgroup tmpfs ro,nosuid,nodev,noexec,mode=755 0 0
cgroup /sys/fs/cgroup/systemd cgroup rw,nosuid,nodev,noexec,relatime,xattr,release_agent=/usr/lib/systemd/systemd-cgroups-agent,name=systemd 0 0
pstore /sys/fs/pstore pstore rw,nosuid,nodev,noexec,relatime 0 0
cgroup /sys/fs/cgroup/cpu,cpuacct cgroup rw,nosuid,nodev,noexec,relatime,cpuacct,cpu 0 0
cgroup /sys/fs/cgroup/cpuset cgroup rw,nosuid,nodev,noexec,relatime,cpuset 0 0
cgroup /sys/fs/cgroup/devices cgroup rw,nosuid,nodev,noexec,relatime,devices 0 0
cgroup /sys/fs/cgroup/perf_event cgroup rw,nosuid,nodev,noexec,relatime,perf_event 0 0
cgroup /sys/fs/cgroup/hugetlb cgroup rw,nosuid,nodev,noexec,relatime,hugetlb 0 0
cgroup /sys/fs/cgroup/net_cls cgroup rw,nosuid,nodev,noexec,relatime,net_cls 0 0
cgroup /sys/fs/cgroup/blkio cgroup rw,nosuid,nodev,noexec,relatime,blkio 0 0
cgroup /sys/fs/cgroup/memory cgroup rw,nosuid,nodev,noexec,relatime,memory 0 0
cgroup /sys/fs/cgroup/freezer cgroup rw,nosuid,nodev,noexec,relatime,freezer 0 0
configfs /sys/kernel/config configfs rw,relatime 0 0
/dev/sda5 / xfs rw,relatime,attr2,inode64,noquota 0 0
systemd-1 /proc/sys/fs/binfmt_misc autofs rw,relatime,fd=33,pgrp=1,timeout=300,minproto=5,maxproto=5,direct 0 0
mqueue /dev/mqueue mqueue rw,relatime 0 0
debugfs /sys/kernel/debug debugfs rw,relatime 0 0
hugetlbfs /dev/hugepages hugetlbfs rw,relatime 0 0
/dev/sdb1 /data1 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdd1 /data3 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdf1 /data5 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdg1 /data6 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdc1 /data2 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sde1 /data4 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda3 /var xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda7 /home xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda1 /boot xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda8 /home/q xfs rw,relatime,attr2,inode64,noquota 0 0
binfmt_misc /proc/sys/fs/binfmt_misc binfmt_misc rw,relatime 0 0
tmpfs /run/user/30303 tmpfs rw,nosuid,nodev,relatime,size=13191248k,mode=700,uid=30303,gid=1001 0 0
/dev/sda8 /home/q/docker/devicemapper xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/mapper/docker-8:8-1610613145-85827f4e55e2febb9f191e507acfb1b32a8ec2f1d281c868f0aca36e12fed2e8 /home/q/docker/devicemapper/mnt/85827f4e55e2febb9f191e507acfb1b32a8ec2f1d281c868f0aca36e12fed2e8 xfs rw,relatime,nouuid,attr2,inode64,logbsize=256k,sunit=512,swidth=1024,noquota 0 0
shm /home/q/docker/containers/601db9d10a356842fbe315d5a2da59514a71217f32f489763b03108b8f80171a/shm tmpfs rw,nosuid,nodev,noexec,relatime,size=65536k 0 0
proc net:[4026532874] proc rw,nosuid,nodev,noexec,relatime 0 0
```
I've no idea about the `net:[4026532874]` mount point means...
> proc net:[4026532874] proc rw,nosuid,nodev,noexec,relatime 0 0

### MountFlags=slave (the origin value before #22806)

the original context in the `/etc/mtab`:

```
rootfs / rootfs rw 0 0
sysfs /sys sysfs rw,nosuid,nodev,noexec,relatime 0 0
proc /proc proc rw,nosuid,nodev,noexec,relatime 0 0
devtmpfs /dev devtmpfs rw,nosuid,size=65946932k,nr_inodes=16486733,mode=755 0 0
securityfs /sys/kernel/security securityfs rw,nosuid,nodev,noexec,relatime 0 0
tmpfs /dev/shm tmpfs rw,nosuid,nodev 0 0
devpts /dev/pts devpts rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000 0 0
tmpfs /run tmpfs rw,nosuid,nodev,mode=755 0 0
tmpfs /sys/fs/cgroup tmpfs ro,nosuid,nodev,noexec,mode=755 0 0
cgroup /sys/fs/cgroup/systemd cgroup rw,nosuid,nodev,noexec,relatime,xattr,release_agent=/usr/lib/systemd/systemd-cgroups-agent,name=systemd 0 0
pstore /sys/fs/pstore pstore rw,nosuid,nodev,noexec,relatime 0 0
cgroup /sys/fs/cgroup/cpu,cpuacct cgroup rw,nosuid,nodev,noexec,relatime,cpuacct,cpu 0 0
cgroup /sys/fs/cgroup/cpuset cgroup rw,nosuid,nodev,noexec,relatime,cpuset 0 0
cgroup /sys/fs/cgroup/devices cgroup rw,nosuid,nodev,noexec,relatime,devices 0 0
cgroup /sys/fs/cgroup/perf_event cgroup rw,nosuid,nodev,noexec,relatime,perf_event 0 0
cgroup /sys/fs/cgroup/hugetlb cgroup rw,nosuid,nodev,noexec,relatime,hugetlb 0 0
cgroup /sys/fs/cgroup/net_cls cgroup rw,nosuid,nodev,noexec,relatime,net_cls 0 0
cgroup /sys/fs/cgroup/blkio cgroup rw,nosuid,nodev,noexec,relatime,blkio 0 0
cgroup /sys/fs/cgroup/memory cgroup rw,nosuid,nodev,noexec,relatime,memory 0 0
cgroup /sys/fs/cgroup/freezer cgroup rw,nosuid,nodev,noexec,relatime,freezer 0 0
configfs /sys/kernel/config configfs rw,relatime 0 0
/dev/sda5 / xfs rw,relatime,attr2,inode64,noquota 0 0
systemd-1 /proc/sys/fs/binfmt_misc autofs rw,relatime,fd=33,pgrp=1,timeout=300,minproto=5,maxproto=5,direct 0 0
mqueue /dev/mqueue mqueue rw,relatime 0 0
debugfs /sys/kernel/debug debugfs rw,relatime 0 0
hugetlbfs /dev/hugepages hugetlbfs rw,relatime 0 0
/dev/sdb1 /data1 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdd1 /data3 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdf1 /data5 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdg1 /data6 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdc1 /data2 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sde1 /data4 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda3 /var xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda7 /home xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda1 /boot xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda8 /home/q xfs rw,relatime,attr2,inode64,noquota 0 0
binfmt_misc /proc/sys/fs/binfmt_misc binfmt_misc rw,relatime 0 0
tmpfs /run/user/30303 tmpfs rw,nosuid,nodev,relatime,size=13191248k,mode=700,uid=30303,gid=1001 0 0
```

then start the container again, still use ubuntu:14.04, check the file:

```
rootfs / rootfs rw 0 0
sysfs /sys sysfs rw,nosuid,nodev,noexec,relatime 0 0
proc /proc proc rw,nosuid,nodev,noexec,relatime 0 0
devtmpfs /dev devtmpfs rw,nosuid,size=65946932k,nr_inodes=16486733,mode=755 0 0
securityfs /sys/kernel/security securityfs rw,nosuid,nodev,noexec,relatime 0 0
tmpfs /dev/shm tmpfs rw,nosuid,nodev 0 0
devpts /dev/pts devpts rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000 0 0
tmpfs /run tmpfs rw,nosuid,nodev,mode=755 0 0
tmpfs /sys/fs/cgroup tmpfs ro,nosuid,nodev,noexec,mode=755 0 0
cgroup /sys/fs/cgroup/systemd cgroup rw,nosuid,nodev,noexec,relatime,xattr,release_agent=/usr/lib/systemd/systemd-cgroups-agent,name=systemd 0 0
pstore /sys/fs/pstore pstore rw,nosuid,nodev,noexec,relatime 0 0
cgroup /sys/fs/cgroup/cpu,cpuacct cgroup rw,nosuid,nodev,noexec,relatime,cpuacct,cpu 0 0
cgroup /sys/fs/cgroup/cpuset cgroup rw,nosuid,nodev,noexec,relatime,cpuset 0 0
cgroup /sys/fs/cgroup/devices cgroup rw,nosuid,nodev,noexec,relatime,devices 0 0
cgroup /sys/fs/cgroup/perf_event cgroup rw,nosuid,nodev,noexec,relatime,perf_event 0 0
cgroup /sys/fs/cgroup/hugetlb cgroup rw,nosuid,nodev,noexec,relatime,hugetlb 0 0
cgroup /sys/fs/cgroup/net_cls cgroup rw,nosuid,nodev,noexec,relatime,net_cls 0 0
cgroup /sys/fs/cgroup/blkio cgroup rw,nosuid,nodev,noexec,relatime,blkio 0 0
cgroup /sys/fs/cgroup/memory cgroup rw,nosuid,nodev,noexec,relatime,memory 0 0
cgroup /sys/fs/cgroup/freezer cgroup rw,nosuid,nodev,noexec,relatime,freezer 0 0
configfs /sys/kernel/config configfs rw,relatime 0 0
/dev/sda5 / xfs rw,relatime,attr2,inode64,noquota 0 0
systemd-1 /proc/sys/fs/binfmt_misc autofs rw,relatime,fd=33,pgrp=1,timeout=300,minproto=5,maxproto=5,direct 0 0
mqueue /dev/mqueue mqueue rw,relatime 0 0
debugfs /sys/kernel/debug debugfs rw,relatime 0 0
hugetlbfs /dev/hugepages hugetlbfs rw,relatime 0 0
/dev/sdb1 /data1 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdd1 /data3 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdf1 /data5 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdg1 /data6 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sdc1 /data2 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sde1 /data4 xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda3 /var xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda7 /home xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda1 /boot xfs rw,relatime,attr2,inode64,noquota 0 0
/dev/sda8 /home/q xfs rw,relatime,attr2,inode64,noquota 0 0
binfmt_misc /proc/sys/fs/binfmt_misc binfmt_misc rw,relatime 0 0
tmpfs /run/user/30303 tmpfs rw,nosuid,nodev,relatime,size=13191248k,mode=700,uid=30303,gid=1001 0 0
```
here is no unusual mount point.

**Comment by dav1x** [comment]
@BrickXu 

I ran into a similar issue. I think it may be a bug in the way the mount point is listed.

Here are my notes:

I figured out why the VM snapshot quiesce isn't working

Essentially, the process for the snapshot quiesce is this:

Lets recap how VMWare VM backups work: a hypervisor sends a signal to vmtoolsd process running inside VM, 'vmtoolsd' freezes all local filesystems, then the snapshots of the disks are made by the hypervisor, after the snapshots are completed another signal is sent to vmtoolsd which unfreezes filesystems previously frozen and all outstanding IO to the previously frozen filesystems is performed. This is a normal scenario.

I enabled debugging and found that the vmtoolsd daemon was referring to a filesystem that didn't exist:

[Jan 30 17:34:57.126] [debug] [vmsvc] SyncDriver: opening path 'net'.
[Jan 30 17:34:57.126] [debug] [vmsvc] SyncDriver: failed to open 'net': 2 (No such file or directory)

The vmtoolsd daemon uses mtab as its possible filesystems to work on. 

[root@ada0fdce3f67 /]# cat /etc/mtab  | grep net
...

proc net:[4026531956] proc rw,nosuid,nodev,noexec,relatime 0 0

But, mount and mtab are showing different information:

[root@ada0fdce3f67 /]# mount | grep net                                                                                                                
...
proc on /host/run/docker/netns/default type proc (rw,nosuid,nodev,noexec,relatime)

[root@ada0fdce3f67 /]# stat /host/run/docker/netns/default
  File: '/host/run/docker/netns/default'
  Size: 0         	Blocks: 0          IO Block: 1024   regular empty file
Device: 3h/3d	Inode: 4026531956  Links: 1
Access: (0444/-r--r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2017-01-30 18:11:42.743924654 +0000
Modify: 2017-01-30 18:11:42.743924654 +0000
Change: 2017-01-30 18:11:42.743924654 +0000
 Birth: -

I manually unmounted /host/run/docker/netns/default and the snapshot quiesce worked fine. 

mtab and mount should be displaying consistent data right?
